control/setup_control.go

```diff
@@ -154,6 +154,7 @@ func (p *PIDLoop) TunePIDLoop(ctx context.Context, cancelFunc context.CancelFunc
 		}
 		// switch sum to depend on the setpoint if position control
 		if p.Options.PositionControlUsingTrapz {
+			p.logger.Info("tuning trapz")
```

Comment:
idk if this was from your own debugging but maybe make it more informative and a .Debug if you want to keep it

Reply:
yeah I added this one to mirror the linear tuning log. can change both

Reply:
yeah I added to linear/angular since there is no way to tell which is which. but this should be fine as debug since there's only one thing being tuned. or maybe something more like "tuning PID" since it's the only one